linkerd/app/outbound/src/http/peer_proxy_errors.rs

```diff
@@ -1,9 +1,5 @@
-use http::{Request, Response};
```

Comment:
we tend not to import Request/Response -- referencing them via, e.g., `http::Request` isn't that much more verbose and is a little more readable -- there's no ambiguity about whether we're dealing with some other type of request/response.